source/_integrations/roomba.markdown

```diff
@@ -21,35 +22,38 @@ This platform has been tested and is confirmed to be working with the iRobot Roo
 
 ## Configuration
 
+To add your Roomba to your installation, go to Configuration >> Integrations in the UI, click the button with + sign and from the list of integrations select iRobot Roomba.
+
 To add your Roomba vacuum to your installation, add the following to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
-vacuum:
-  - platform: roomba
-    host: IP_ADDRESS_OR_HOSTNAME
-    username: BLID
+roomba:
+  - host: IP_ADDRESS_OR_HOSTNAME
+    blid: BLID
     password: PASSWORD
+    ...
```

Comment:
Only required config variables in the first example, but you can add a more advanced example at the bottom.